autopush/web/base.py

```diff
@@ -335,6 +335,14 @@ def _router_fail_err(self, fail, router_type=None, vapid=False, uaid=None):
         # reporting a not found and they're mobile.
         if exc.status_code in [404, 410] and router_type in [
                 'apns', 'fcm', 'adm']:
+            tags = self._base_tags.update({
```

Comment:
`update` mutates the dictionary and returns `None`:
https://docs.python.org/3/library/stdtypes.html#dict.update

This is probably not the behavior you're expecting?

Reply:
crap. You're right. I always get that wrong. Thanks!